OasisDEX/oasis-borrow
# Stop-loss for AAVE validation refactor

# Stop-loss for AAVE validation refactor

Build is failing - need to know wtf this entire metadata mechanism requires to be declared twice, separately for manage vault and for open vault.
  
## Changes 👷‍♀️

- Created repository of stand alone methods for errors and warnings validations for Stop-Loss - same thing needs to happen for all other automation features, but it's going to be a lot easier now,
- changed Stop-loss metadata interface structure in places related to validation.
  
## How to test 🧪

- Try to trigger some errors,
- or edit validations methods to return `true` by default to see if whole mechanism of passing data works.
